app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,56 @@
+@charset "utf-8";
+
+@import "normalize-rails";
+@import "bourbon";
+@import "base/grid-settings";
+@import "neat";
+@import "base/base";
+@import "refills/flashes";
+
+$binding-site: #f00;
+$container-background: #eee;
+$footer-color: #bbb;
+$target-region: #00f;
+
+$radius: 5px;
+
+#Container {
```

Comment:
Avoid using id selectors<br>Selector `Container` should be written in lowercase with hyphens